packages/x-components/static-docs/build-search-ui/README.md

```diff
@@ -20,7 +20,7 @@ Start building your search and discovery experience:
 
 ::: slot right-intro-column
 
-<img :src="$withBase('/assets/media/interface/build-search-ui.svg')" alt="Build Your Search UI">
+<img :src="$withBase('/assets/x/interface/build-search-ui.svg')" alt="Build Your Search UI">
```

Comment:
Just curious, why some images are referenced through an `img` tag with a `$withBase` helper, but others are  used with a markdown image and a `~` path?